src/app/zones/views/IndexView.vue

```diff
@@ -238,6 +264,59 @@ const isDeleteModalVisible = ref(false)
 const isCreateZoneButtonVisible = ref(false)
 const deleteZoneName = ref('')
 
+type Gresses<T> = Record<string, {online: T[], offline: T[]}>
```

Comment:
ZoneProxy is a likely better name than `Gresses`

Reply:
😆 ah yes definitely! thankyou!